src/test/java/jek/collections/IntArrayListTest.java

```diff
@@ -7,7 +7,14 @@
 class IntArrayListTest {
 
     @Test
-    void size() {
+    void sizeShouldBeZero() {
+        IntArrayList ial = new IntArrayList();
```

Comment:
Each of the methods begin with the same line. Please read about the @Setup, @Before and other related annotations and use them to write method that will initialize the array beforehand.